src/test/datascience/notebook/executionService.vscode.test.ts

```diff
@@ -123,6 +123,14 @@ suite('DataScience - VSCode Notebook - (Execution) (slow)', function () {
 
         await Promise.all([runCell(cell), waitForTextOutput(cell, '123412341234')]);
     });
+    test('Test __vsc_ipynb_file__ defined in cell using VSCode Kernel', async () => {
```

Comment:
It looks like this is failing in remote (make sense since we don't run the kernel startup code in remote).

Can you add this to the test?

```
        if (IS_REMOTE_NATIVE_TEST) {
            return this.skip();
        }
```
And change the function from ```async () =>``` to ```async function()```

Reply:
That should skip this test on remote.